content/post/data-analyst.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "The Analyst career path"
```

Comment:
Title-casing is odd here

Reply:
And also maybe not a great title for this post -- something like "The many hats of a data analyst" conveys better what's actually discussed here